test/e2e/cypress/e2e/settings.cy.js

```diff
@@ -806,4 +807,71 @@ context('Settings page', () => {
       });
     });
   });
+
+  describe('Forbidden actions', () => {
+    const password = 'password';
+
+    beforeEach(() => {
+      cy.deleteAllUsers();
+      cy.logout();
+      const user = createUserRequestFactory.build({
+        password,
+        password_confirmation: password,
+      });
+      cy.wrap(user).as('user');
+    });
+
+    it('should enable settings buttons if the user has the correct abilities', () => {
+      const userAbilites = [
+        { name: 'all', resource: 'activity_logs_settings' },
+        { name: 'all', resource: 'api_key_settings' },
+        { name: 'all', resource: 'suma_settings' },
+      ];
+      cy.get('@user').then((user) => {
+        cy.createUserWithAbilities(user, userAbilites);
+        cy.login(user.username, password);
+      });
+      cy.visit(`/settings`);
+      // API Key settings button
+      cy.contains('button', 'Generate Key')
+        .should('be.visible')
+        .and('be.enabled');
+      // SUSE Manager config settings button
+      cy.contains('button', 'Test Connection').should('be.enabled');
+      cy.contains('h2', 'SUSE Manager Config')
+        .next()
+        .contains('button', 'Edit Settings')
+        .should('be.enabled');
+      cy.contains('button', 'Clear Settings').should('be.enabled');
+      // Activity Logs settings button
+      cy.contains('h2', 'Activity Logs')
+        .next()
+        .contains('button', 'Edit Settings')
+        .should('be.enabled');
+      cy.contains('button', 'Clear Settings').should('be.enabled');
```

Comment:
`cy.contains('button', 'Clear Settings').should('be.enabled');` this line is duplicated